docs/playground/fr/3-7/Syntax and Messaging/Optional Chaining.ts

```diff
@@ -1,15 +1,14 @@
 //// { compiler: {  }, order: 1 }
 
-// Optional chaining reached TC39 Stage 3 consensus during
-// 3.7's development. Optional Chaining allows you to write
-// code which can immediately stop running expressions when
-// it hits a null or undefined.
+// Le chaînage optionnel a atteint l'étape 3 du TC39 pendant le development
+// de la version 3.7. Le chaînage optionnel permet d'écrire du code qui va
+// interrompre l'execution des expressions dès qu'il atteint une valeur
+// null ou undefined.
```

Comment:
I decided to keep primitive types like "null", "undefined" and "false" in english, as they would be anyway used verbatim in the code. If that makes sense it could be added to the list of terms we don't translate.

Reply:
Looks good to me. Tho I'd probably put the `null` and `undefined` values in preformatted blocks (by surrounding them with backticks). I don't really know if it's possible, like in Markdown.

We could totally add them to the list of words that shouldn't be translated, with all the other JavaScript/TypeScript keywords. But do we have such a list in the first place?

Reply:
We have this discussion to list those: https://github.com/microsoft/TypeScript-Website-Localizations/discussions/108

Yeah i like the code formatted block but as they are not used in the en version i didn't want to deviate here and cause any issues. Could be also single / double quotes just to make them stand out..